user/enterprise/setting-up-travis-ci-enterprise.md

```diff
@@ -0,0 +1,162 @@
+Enterprise worker. We recommend using AWS' `c4.2xlarge` instance running
+Ubuntu 16.04 LTS as the underlying operating system.
+
+1. *On your virtual machine management platform*, create a Travis CI Platform Security Group.
+
+    If you're setting up your AMI for the first time you need to create
+    a Security Group. From the EC2 management console, create an entry for
+    each port in the table below:
+
+    | Port | Service         | Description                                                                  |
+    |:-----|:----------------|:-----------------------------------------------------------------------------|
+    | 8800 | Custom TCP Rule | This port is to access the admin dashboard for your Enterprise installation. |
+    | 5672 | Custom TCP Rule | For RabbitMQ Non-SSL.                                                        |
+    | 4567 | Custom TCP Rule | For RabbitMQ SSL.                                                            |
+    | 443  | HTTPS           | Web application over HTTPS access.                                           |
+    | 80   | HTTP            | Web application access.                                                      |
+    | 22   | SSH             | SSH access.                                                                  |
+
+1. If you're using a hostname and not just an IP address, this would be a time to configure it.
+
+1. *On your new virtual machine*, download and run the installation script:
+
+
+    ```
+    curl -sSL -o /tmp/installer.sh https://enterprise.travis-ci.com/install
+    sudo bash /tmp/installer.sh
+    ```
+
+3. *In your browser*, navigate to `https://<hostname>:8800` (your Enterprise
+installation's hostname, port 8800) to complete the setup:
+
+    a. Add a secure certificate / configure a trusted one [HTTPS Replicated page]  
+    b. Upload your license - [Validaing license file]
+    c. Configure access to the Admin Console (password / openldap)
+    d. Connect your GitHub Enterprise or GitHub.com with Travis CI enterprise -
+    e. Optionally, configure Email, Metrics, Caches
+    f. Get the *RabitMQ password* for the Worker setup
+
+
+## 2. Setting up the Enterprise Worker virtual machine
+
+The Travis CI Enterprise Worker manages build containers and reports build
+statuses back to the platform. It must be installed on a separate machine
+instance from the Platform. We recommend using AWS' `c4.2xlarge` instance running
+Ubuntu 16.04 LTS (beta) as the underlying operating system.
+
+
+
+* Make sure you have already [set up the Enterprise Platform](/user/enterprise/setting-up-travis-ci-enterprise/#1-setting-up-the-travis-ci-enterprise-platform-virtual-machine) and have the *RabbitMQ password* and the *hostname* from the Platform Dashboard.
+* If this is the first time you're setting up a worker machine with Trusty build images, please enable [this feature flag](/user/enterprise/trusty#enabling-the-trusty-beta-feature-flag) on your platform machine.
```

Comment:
This bullet point is confusing since we are telling the user above to use Xenial and not trusty.

Reply:
This feature flag does not apply anymore. That was mandatory on Enterprise 2.1 but doesn't anymore for 2.2 and later.